<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SmartLogic One :: Control GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);
color: rgb(255, 255, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 64, 74);
color: rgba(255, 255, 255, 224);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCrete_new_config"/>
    <addaction name="actionLoad_config"/>
    <addaction name="actionSave_active_config"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuViewStyle">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionStyleDark"/>
     <addaction name="actionStyleLight"/>
    </widget>
    <widget class="QMenu" name="menuWindows">
     <property name="title">
      <string>Windows</string>
     </property>
     <addaction name="actionDATA_dock_window"/>
     <addaction name="actionINFO_dock_window"/>
     <addaction name="actionCONTROL_dock_window"/>
    </widget>
    <addaction name="menuViewStyle"/>
    <addaction name="menuWindows"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 44, 47);</string>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSETTINGS">
   <property name="minimumSize">
    <size>
     <width>84</width>
     <height>48</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>SETTINGS</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="DockModulInfo_VLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSENSORS">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>84</width>
     <height>46</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(97, 134, 100, 148);</string>
   </property>
   <property name="windowTitle">
    <string>SENSORS</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="DockModulControl_VLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDATA">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>42</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(128, 150, 150, 168);</string>
   </property>
   <property name="windowTitle">
    <string>DATA</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolBar_ALL_MODULES"/>
   <addaction name="separator"/>
   <addaction name="actionToolBar_SCENARIES"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOUTS">
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(168, 64, 60, 224);</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>ACTUATORS</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="DockModulScenaries_VLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCrete_new_config">
   <property name="text">
    <string>Crete new config</string>
   </property>
  </action>
  <action name="actionLoad_config">
   <property name="text">
    <string>Load config</string>
   </property>
  </action>
  <action name="actionSave_active_config">
   <property name="text">
    <string>Save active config</string>
   </property>
  </action>
  <action name="actionStyleDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionStyleLight">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionToolBar_ALL_MODULES">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MODULES</string>
   </property>
   <property name="toolTip">
    <string>MODULES</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionToolBar_SCENARIES">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SCENARIES</string>
   </property>
   <property name="toolTip">
    <string>SCENARIES</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDATA_dock_window">
   <property name="text">
    <string>DATA  dock window</string>
   </property>
  </action>
  <action name="actionINFO_dock_window">
   <property name="text">
    <string>INFO  dock window</string>
   </property>
  </action>
  <action name="actionCONTROL_dock_window">
   <property name="text">
    <string>CONTROL dock window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionINFO_dock_window</sender>
   <signal>triggered()</signal>
   <receiver>dockWidgetSETTINGS</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDATA_dock_window</sender>
   <signal>triggered()</signal>
   <receiver>dockWidgetDATA</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCONTROL_dock_window</sender>
   <signal>triggered()</signal>
   <receiver>dockWidgetSENSORS</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
